client/extensions/woocommerce/app/promotions/fields/checkbox-field.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { omit } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import FormCheckbox from 'components/forms/form-checkbox';
+import FormField from './form-field';
+
+const CheckboxField = ( props ) => {
+	const { fieldName, explanationText, placeholderText, value, edit } = props;
+	const renderedValue = ( 'undefined' !== typeof value ? value : false );
+
+	const onChange = () => {
+		edit( fieldName, ! value );
+	};
+
+	return (
+		<FormField { ...omit( props, 'explanationText' ) } >
+			<FormCheckbox
+				id={ fieldName + '-label' }
+				aria-describedby={ explanationText && fieldName + '-description' }
+                checked={ renderedValue }
```

Comment:
Indentation.

Reply:
Thanks, I this VS Code used spaces instead of tabs for some weird reason, but I re-formatted it in VS Code and it used tabs this time, not sure what that's about.